src/server/service/achievements.ts

```diff
@@ -28,13 +29,39 @@ const lastMessageAchievement: MatchAchievement = {
   },
 };
 
+const perfectScoreAchievement: MatchAchievement = {
+  id: "12",
+  name: "Perfect score",
+  description: "Get all votes correct in a match",
+  calculate: ({ botsBusted, otherPlayers }) => {
+    const agents = otherPlayers.filter((p) => p.isBot);
+
+    if (botsBusted !== agents.length) return 0;
+
+    return 13;
+  },
+};
+
+const someoneSelectedYouAsABotAchievement: MatchAchievement = {
+  id: "13",
+  name: "Someone selected you as a bot",
+  description: "Someone selected you as a bot",
+  calculate: ({ otherPlayers, player }) => {
+    let isVotedAgainst = 0;
```

Comment:
This way to describe the logic is a bit confusing. Let's be more clear about those 13 points. You can return a boolean from the conditional and return the right amount of points. 
The idea is that all the 'calculate' look similar on the return. So it's easier to identify what's going on in a glance